domain/modeldefaults/bootstrap/bootstrap_test.go

```diff
@@ -34,8 +30,8 @@ func (_ *bootstrapSuite) TestBootstrapModelDefaults(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(defaults["foo"].Value, gc.Equals, "region")
 	c.Assert(defaults["foo"].Source, gc.Equals, "region")
-	c.Assert(defaults["default"].Value, gc.Equals, "some value")
-	c.Assert(defaults["default"].Source, gc.Equals, "default")
+	c.Assert(defaults["image-stream"].Value, gc.Equals, "released")
+	c.Assert(defaults["image-stream"].Source, gc.Equals, "default")
```

Comment:
We should check here that some of the default keys are set e.g. `lxd-snap-channel`.

Reply:
We are with "image-stream".

Reply:
Ok, maybe just add a comment to that effect ("check that default config values are set") and test a few more values are in there.

Reply:
Have updated the tests to check for all of the defaults.